spec/lib/bank_account/account_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe BankAccount::Account do
+  it "responds to the .new class method" do
+    expect(BankAccount::Account).to(respond_to(:new))
+  end
+end
+
+describe "Declining to Open an Account with Less than zero" do
```

Comment:
You can think about it this way organizationally:

describe : subject
context : action
it : result